pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs

```diff
@@ -14,7 +14,19 @@ $(function() {
                     table.data('filter-student-only', filterOn);
 
                     $('.columns button[name=studentsOnly]').toggleClass('active', filterOn);
-                    table.bootstrapTable('filterBy', filterOn ? { role: 'Student' } : {});
+                    <% if (assessment.group_work) { %>
+                        table.bootstrapTable('filterBy', {id: 1}, {
```

Comment:
why is id there?

Reply:
For the `filterBy` method, it seems that a `filter` has to be specified in the parameters (link [here](https://bootstrap-table.com/docs/api/methods/#filterby)). If the `filter` parameter is not specified, the filter does not do anything, and no roles will be filtered out. I added a comment that will hopefully clarify things a little, and also renamed some variable names for clarity.

Reply:
can you set the `filter` to a blank object (`{}`)? I haven't tested, just wondering.

Reply:
Setting the filter to a blank object will not filter out any rows.

Reply:
What about `group_roles: "Student"`? Since it has no effect, at least keep it with a meaningful value...

Reply:
Alright, let's do that. I'll keep the comments to clarify too.

Reply:
It looks as though this is fixed in the latest version of `bootstrap-table` via https://github.com/wenzhixin/bootstrap-table/pull/6104. @jonatanschroeder do you know of any reasons we shouldn't install the latest version?

Reply:
I don't have any specific reason, I'd just like to make sure we test it properly before upgrading bootstrap-table. The last time we upgraded it (to 1.19.1) it introduced some weird errors (see #5362), and bootstrap-table functionality isn't covered by our CI tests, so I'd prefer to do it in a separate PR (dependabot has #5794) with proper human testing.